### PR TITLE
Fix for Report section displays date pickers in Chinese

### PR DESCRIPTION
**Pull request by Espurna**
## Description of the issue/feature this PR addresses

Linked issue: https://github.com/senaite/senaite.core/issues/1633

## Current behavior before PR

Issue explained here: https://github.com/senaite/senaite.core/pull/1570/

## Desired behavior after PR is merged

"Date picker 2 months" in reports uses the proper language.

--
I confirm I have tested this PR thoroughly and coded it according to [PEP8][1]
and [Plone's Python styleguide][2] standards.

[1]: https://www.python.org/dev/peps/pep-0008
[2]: https://docs.plone.org/develop/styleguide/python.html

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -10,6 +10,7 @@ Changelog
 - #1628 Fix instrument import for analyses with result options
 - #1625 Fix assignment of analyses via worksheet template when Worksheet is full
 - #1617 Fix writing methods on read when reindexing services
+- #1633 Fix report section displays date pickers in Chinese
```
Comment (xispa) [inline CHANGES.rst]: @Espurna always add the changelog entry to the top of the list. They are always sorted from newest to oldest, as versions.

**Review by xispa — CHANGES_REQUESTED** [review]
It does work to me..... in which report you see this problem?

**Comment by Espurna** [comment]
> It does work to me..... in which report you see this problem?

To all the reports.

**Comment by xispa** [comment]
@Espurna I've done some cleanup. Can you confirm date pickers are working properly in your instance?

**Comment by Espurna** [comment]
@xispa After you changes, it is still working well.

**Review by xispa — APPROVED** [review]

**Review by xispa — APPROVED** [review]

**Pull request merged**